composer.json

```diff
@@ -29,21 +29,28 @@
         "friends-of-behat/symfony-extension": "^2.1",
         "friends-of-behat/variadic-extension": "^1.3",
         "friendsofsymfony/oauth-server-bundle": "^1.6 || >2.0.0-alpha.0 ^2.0@dev",
+        "j13k/yaml-lint": "^1.1",
         "lakion/mink-debug-extension": "^2.0.0",
+        "php-parallel-lint/php-parallel-lint": "^1.2",
+        "phpmd/phpmd": "^2.8",
+        "phpro/grumphp": "^1.1.0",
         "phpspec/phpspec": "^7.0",
         "phpstan/extension-installer": "^1.0",
         "phpstan/phpstan": "0.12.74",
         "phpstan/phpstan-doctrine": "0.12.31",
         "phpstan/phpstan-strict-rules": "^0.12.0",
         "phpstan/phpstan-webmozart-assert": "0.12.12",
         "phpunit/phpunit": "^9.5",
-        "sensiolabs/security-checker": "^6.0",
-        "sylius-labs/coding-standard": "^3.1",
+        "povils/phpmnd": "^2.2",
+        "seld/jsonlint": "^1.7",
+        "sylius-labs/coding-standard": "^3.0",
```

Comment:
Can you : 
* use sylius-labs/coding-standard v4
* remove symplify/easy-coding-standard just below
* config ecs with PHP ex : https://github.com/synolia/SyliusAkeneoPlugin/blob/master/ruleset/ecs.php

Reply:
I did it cause, sylius-plugin skeletton asked for the sylius-labs/coding-standard 3 but i can update it ! :smiley: